db/migrate/20241113205619_add_fed_income_total_to_nj_intake.rb

```diff
@@ -0,0 +1,5 @@
+class AddFedIncomeTotalToNjIntake < ActiveRecord::Migration[7.1]
+  def change
+    add_column :state_file_nj_intakes, :fed_income_total, :integer
```

Comment:
Same question as above-- should this maybe go on base intake?

Reply:
was actually removed from db altogether and is just in the `direct_file_data`